Sources/XCHammer/XCHammerGenerateOptions.swift

```diff
@@ -35,7 +35,7 @@ struct XCHammerGenerateOptions {
     }
 
     var pathsSet: Set<String> {
-        let paths = config.projects[projectName]?.paths ?? []
+        let paths = config.projects[projectName]?.paths ?? ["**"]
```

Comment:
@jerrymarino any issues if two projects have overlapping globs? Would they duplicate the sources ?

Reply:
It `filters` targets and consequently files based on these paths - if the user wants to do that it will work 👍